domain/src/main/java/co/nimblehq/compose/crypto/domain/repository/CoinRepository.kt

```diff
@@ -3,6 +3,7 @@ package co.nimblehq.compose.crypto.domain.repository
 import co.nimblehq.compose.crypto.domain.model.CoinItem
 import kotlinx.coroutines.flow.Flow
 
+@Suppress("LongParameterList")
 interface CoinRepository {
     fun getTrendingList(
```

Comment:
Actually, we should put this `LongParameterList` Suppress at method only

```
    @Suppress("LongParameterList")
    fun getTrendingList(
```

Reply:
@luongvo I want to add it here to skip the same warning on Repository 😄 We could ignore long parameter on this layer, right? 😄

Reply:
@hoangnguyen92dn Normally, I don't want to ignore almost of issues at class level, except class's issues such as `LargeClass`, etc. Ignoring them at class level makes me feel like we can ignore them from the config instead. 🤷‍♂️